### ISSUE TITLE
The "Chest" string is not displayed above the Chest.

### ISSUE THREAD

**Issue by Quintus24** [AmProsius/gothic-1-community-patch#281]
Hello, everyone!

I would like to report the following bug:

**Describe the bug:**
The issue can be encountered in the **Free Mine** if the user interacts with a **Chest**.
The **Chest** is correctly highlighted when the user faces it, however, the "**Chest**" string is not displayed.
The **Chest** is hidden under a catwalk, near the **FMC_HUT11_OUT** waypoint.

**Expected behavior:**
The "**Chest**" string is correctly displayed when the **Chest** is interacted with or highlighted.

**Steps to reproduce the issue:**
1. Travel to the "**Free Mine**" and reach the **FMC_HUT11_OUT** waypoint.
2. Find the hidden **Chest** and highlight/interact with it.
3. Notice the "**Chest**" string is not displayed.

**Additional context**
Minor issue, the chest is empty anyway, not sure if that part is intended.
Could be that the string is displayed but it's obscured by the environment.

_Attaching some screenshots for more clear information_.

1. Location at **FMC_HUT11_OUT** waypoint.
![Location @ FMC_HUT11_OUT](https://user-images.githubusercontent.com/93062373/148294130-af3c4fb4-2b52-4e90-a717-51355121f035.JPG)

2. Chest not highlighted.
![Chest not highlighted](https://user-images.githubusercontent.com/93062373/148294185-8466cd38-6b11-4357-b2c3-36d869f7d292.JPG)

3. Chest is highlighted.
![Chest is highlighted](https://user-images.githubusercontent.com/93062373/148294210-1cbd2923-9f1a-4588-a7a8-8be735623ecf.JPG)

4. Interacting with the Chest.
![Free Mine Chest](https://user-images.githubusercontent.com/93062373/148294256-ccc8e50f-4847-409d-8348-d69a720ca941.JPG)

Thank You and Best Regards!
Quintus24

**Comment by szapp** [comment]
I would agree, that the focus name is just occluded. Since the chest is empty, it may have been intended to be a decorative VOB (given that it just wasn't overlooked when filling containers with content). I would suggest the following options:

1. Either leave everything as it is, as it is only a minor inconvenience.
2. Move the chest in such a way that the focus name becomes visible. This risks making it worse, if the world mesh has been changed in that area.
3. Check if the chest is empty and replace the MOB by a VOB of the same visual. This would make it purely decorative, without any option to interact with it.
